cudax/include/cuda/experimental/__stf/internal/cuda_kernel_scope.cuh

```diff
@@ -37,6 +37,28 @@ namespace cuda::experimental::stf
 class graph_ctx;
 class stream_ctx;
 
+namespace reserved
```

Comment:
Important: if this is an implementation detail namespace, please name it `detail`